Resources/Prototypes/Catalog/VendingMachines/Inventories/robodrobe.yml

```diff
@@ -11,3 +11,6 @@
     ClothingHeadsetRobotics: 2
     ClothingOuterWinterRobo: 2
     ClothingShoesBootsWinterRobo: 2 #Delta V: Add departmental winter boots
+  contrabandInventory: #Delta V: Added Techpriest Robe to Contra Inventory
```

Comment:
Why is this in the contrabandInventory as opposed to the regular inventory?

Reply:
To put it simply, just so the robes aren't immediately accessible, so its more like a small secret for tech cult-ish roleplay, IE worshipping machines and the like. Can't just hand them the spooky robes after all

Reply:
Would you be potentially opposed to putting these in the Autodrobe instead?

Reply:
The costume machine? Maybe, just thought it would be fitting to hide it in the robodrobe, cause it feels thematic. But if you want i can toss these into there instead

Reply:
Eh, I'd prefer it a bit more. I personally think robes fit more within theatre than in science, but hey. It is your PR and honestly I'm not actually super fussed about any of that yet. Epistemics is one of the later departments I plan on touching up.

Reply:
For now, I think I will toss them behind the contra inventory of the Autodrobe. I f you want your spooky techy robes you gotta do a little work at least